securedrop/tests/migrations/migration_b58139cfdc8c.py

```diff
@@ -182,7 +198,10 @@ def load_data(self):
 
     def check_downgrade(self):
         '''
-        Verify that the checksum column is now gone.
+        - Verify that the checksum column is now gone.
+        - Verify that the migration worked by checking that the journalist added in load_data has
```

Comment:
This line of the docstring is wrong. On the downgrade the `first_name` and `last_name` columns won't exist. I think this was meant for `check_upgrade`.